diff-match-patch/diff-match-patch.d.ts

```diff
@@ -31,6 +38,12 @@ declare module "diff-match-patch" {
         diff_levenshtein(diffs: Diff[]): number;
         diff_toDelta(diffs: Diff[]): string;
         diff_fromDelta(text1: string, delta: string): Diff[];
+
+        patch_make(text1: any, text2?: string): Patch[];
```

Comment:
looks like this method can take up to three parameters. can you add overloads to describe the [4 ways](https://github.com/ForbesLindesay/diff-match-patch/blob/master/index.js#L1616-L1628) to call this method?